packages/components/src/components/accordion-item/accordion-item.lite.tsx

```diff
@@ -54,9 +54,9 @@ export default function DBAccordionItem(props: DBAccordionItemProps) {
 			open={state._open}
 			name={props.name}>
 			<summary onClick={(event) => state.toggle(event)}>
-				<Show when={props.title}>{props.title}</Show>
-				<Show when={!props.title}>
-					<Slot name="title" />
+				<Show when={props.headline}>{props.headline}</Show>
+				<Show when={!props.headline}>
+					<Slot name="headline" />
 				</Show>
```

Comment:
```suggestion
				<Slot name="headline">{props.headline}</Slot>
```

Wouldn't this work as well with Mitosis? At least I think to remember this by Web Components slots. It doesn't do exactly the same, doesn't check for the `props.headline` property, but is a little simpler.

Reply:
It's a little bit confusing...
At the moment Mitosis uses `slotHeadline` for React which isn't the case for new versions. I changed it already in the [beta branch](https://github.com/db-ui/mono/blob/feat-color-palette/packages/components/src/components/accordion-item/model.ts#L28).

Is your solution working?

Reply:
> Is your solution working?

I haven't tried it, I wanted to either ask for your experience/knowledge or trigger your curiosity.

Reply:
` <ng-content> element cannot have content. [plugin angular-compiler]

    ../../output/angular/src/components/accordion-item/accordion-item.ts:28:8:
      28 │         <ng-content select="[headline]">{{headline}}</ng-content>
`

Nope it's not working for angular

Reply:
okay. Let's move to Web Components.